cfme/utils/template/ec2.py

```diff
@@ -0,0 +1,138 @@
+import os
+import re
+import urllib2
+
+from cfme.utils.log import logger
+from cfme.utils.template.base import BaseTemplateUpload, log_wrap
+from cfme.utils.wait import wait_for
+
+
+class EC2TemplateUpload(BaseTemplateUpload):
+    log_name = 'EC2'
+    provider_type = 'ec2'
+    image_pattern = re.compile(r'<a href="?\'?([^"\']*ec2[^"\'>]*)')
+
+    def get_creds(self, creds_type=None, **kwargs):
```

Comment:
Looks like this is unused here, and there's no more need for the aws_cli_tool_client query because the file is downloaded locally and uploaded through the provider's wrapanapi class.

Reply:
So I can change script to download file locally instead of aws_client_tool?

Reply:
`download_image` method is already pulling the file to a local directory :)

But yep, no more need for aws_client_tool since this is strictly using the wrapanapi mgmt class to interact with the API.